api/src/middleware/authorization-middleware.ts

```diff
@@ -0,0 +1,91 @@
+import { type NextFunction, type Response } from "express"
+import { type Request as JwtRequest } from "express-jwt"
+import { isNil } from "lodash"
+
+import logger from "@/utils/logger"
+import { User } from "@/models"
+
+import auth0Integration, { Auth0PayloadError } from "@/integrations/auth0-integration"
+
+import NodeCache from "node-cache"
```

Comment:
I generally try to follow Python import ordering see https://icefoganalytics.slack.com/archives/C02URJHPFTQ/p1714515678853629